code/controllers/subsystem/processing/fluids.dm

```diff
@@ -3,3 +3,22 @@ PROCESSING_SUBSYSTEM_DEF(fluids)
 	wait = 20
 	stat_tag = "FD" //its actually Fluid Ducts
 	flags = SS_NO_INIT | SS_TICKER
+
+	///We randomly pick a duct from this list to start building a network
+	var/list/prime_queued = list()
+	///We track all the ducts that are currently building and queued in the timer subsystem, so we only prime new ductnetworks
+	var/list/build_queued = list()
```

Comment:
Could this be achieved with a var on the pipe? doing lookups in large lists tends to be expensive